src/components/modal/content/ES/parts/GPL.jsx

```diff
@@ -0,0 +1,47 @@
+/** @jsx h */
+import { h } from 'preact';
+import { useContent, useProductMeta } from '../../../lib';
+
+const auCurrencyFormat = string => string.replace(/\.00/g, '');
```

Comment:
```suggestion
const esCurrencyFormat = string => string.replace(/\.00/g, '');
```
and update lines 17 and 39 with the renamed variable